MovieApp/PresentationLayer/Favorites/FavoritesViewController.swift

```diff
@@ -11,3 +16,68 @@ class FavoritesViewController: UIViewController {
     }
 
 }
+
+extension FavoritesViewController: UICollectionViewDataSource {
+
+    func collectionView(
+        _ collectionView: UICollectionView,
+        numberOfItemsInSection section: Int
+    ) -> Int {
+        100
+    }
+
+    func collectionView(
+        _ collectionView: UICollectionView,
+        cellForItemAt indexPath: IndexPath
+    ) -> UICollectionViewCell {
+        guard
+            let cell = movieCollectionView.dequeueReusableCell(
+                withReuseIdentifier: MoviePosterCell.cellIdentifier,
+                for: indexPath) as? MoviePosterCell
+        else {
+            let cell = MoviePosterCell()
+            return cell
+        }
+
+        let movie = MovieViewModel(
+            id: 1,
+            about: "",
+            name: "",
+            posterSource: "https://image.tmdb.org/t/p/w185/rYFAvSPlQUCebayLcxyK79yvtvV.jpg",
+            genres: [],
+            isFavorited: true)
+        
+        cell.setData(for: movie)
+        cell.moviePoster.onFavoriteToggle = { _ in }
+        return cell
+    }
+
+}
+
+extension FavoritesViewController: UICollectionViewDelegateFlowLayout {
+
+    func collectionView(
+        _ collectionView: UICollectionView,
+        layout collectionViewLayout: UICollectionViewLayout,
+        sizeForItemAt indexPath: IndexPath
+    ) -> CGSize {
+        CGSize(width: 105, height: 155)
```

Comment:
Not really sure this is correct. How does it look on bigger screens?

Reply:
I thought it was a bit suspicious as well but I decided to follow the Figma design.
Looks a bit spaced out. Here's a comparison:
[2nd gen iPhone SE](https://user-images.githubusercontent.com/53893982/127853044-09e33b5b-4b4e-49c2-8d7e-c9b0f851a9bd.png)
[iPhone 12 Pro Max](https://user-images.githubusercontent.com/53893982/127853237-2099fb5c-cab8-440b-87f1-44886ea99eb7.png)

Reply:
OK, leave it for now and we will return to it later